docs/collecting-data/collecting-from-own-applications/mobile-trackers/hybrid-apps/index.md

```diff
@@ -14,6 +14,6 @@ This feature is available since v4.
 
 Hybrid apps are mobile apps that in addition to a native interface, provide part of the UI through an embedded Web view. Snowplow events are tracked from both the native code (e.g., written in Swift or Kotlin) as well as the Web view (in JavaScript). Our goal is to have both events tracked from the native code as well as the Web view share the same session and appear as tracked with the same tracker.
 
-This use case is supported by implementing the [Snowplow WebView tracker](https://github.com/snowplow-incubator/snowplow-webview-tracker) in the Web view in your app. The WebView tracker is able to pass events to the iOS tracker which sends them to the collector.
+This use case is supported by implementing the [Snowplow WebView tracker](../../webview-tracker/index.md) in the Web view in your app. The WebView tracker is able to pass events to the iOS/Android tracker which sends them to the collector.
 
-Please refer to the [Snowplow Hybrid Apps Tracking tutorial](https://snowplow-incubator.github.io/snowplow-hybrid-apps-accelerator/) for a step-by-step guide how to set up tracking in hybrid apps.
+Please refer to the [Snowplow Hybrid Apps Tracking tutorial](https://docs.snowplow.io/accelerators/hybrid) for a step-by-step guide how to set up tracking in hybrid apps.
```

Comment:
Should say accelerator rather than tutorial I think?

Wondering if some headings on this page would be good too. It's only short but a "WebView Tracker" and "Mobile & Hybrid Accelerator" heading above paragraph 2 and 3 respectively might draw some focus vs the block of text?

Reply:
Good idea, changed.